offline-geocode/src/main/res/layout/searchview.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/linearLayout1"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="15dp"
+    android:layout_marginRight="15dp"
+    android:layout_marginTop="20dp"
+    android:background="#ffff"
```

Comment:
@doneill I tested by removing the color background on LinerLayout and launched the app, the background was transparent. After that I tried to set a white background to both the  Spinner and SearchView, in this case, Spinner's dropdown icon was also painted white. Hence we will need to keep the background in LinerLayout.